src/main/webapp/scripts/appscript.js

```diff
@@ -64,12 +60,63 @@ async function populateAccountList() {
  * the readfile page.
  */
 async function addConfigToPage() {
-  // TODO: Add query string support to sent to servlet.
-  var vendorID = document.getElementById('customer-ids');
-  
+  const queryString = buildQueryString();
   // Fetch the json configuration and format it to print on the page. 
-  fetch('/BillingConfig')
+  fetch(queryString)
     .then(response => response.json())
     .then(data => document.getElementById('json-text').innerText = 
     JSON.stringify(data, undefined, 4));
 }
+
+/**
+ * Build the edit form's action attribute and populates text from existing configuration
+ */
+function buildEditForm() {
+  const editForm = document.getElementById('edit-config-form');
+  editForm.action = buildQueryString();
+}
+
+function buildQueryString() {
+  const selectedVendorId = document.getElementById('customer-ids').value;
+  const selectedAccountId = document.getElementById('account-ids').value;
+  return `/BillingConfig?vendorID=${selectedVendorId}&accountID=${selectedAccountId}`;
+}
+
+/**
+ * Return the form validity. If all checks pass, this function returns true and allows the post request
```

Comment:
I would just say "Validates the fields in the edit form input and returns true iff all fields have a valid format"